criu/include/memfd-secret.h

```diff
@@ -0,0 +1,20 @@
+#ifndef __CR_MEMFD_SECRET_H__
+#define __CR_MEMFD_SECRET_H__
+
+#include <sys/types.h>
+#include "common/config.h"
+
+extern int is_memfd_secret(dev_t dev);
+extern const struct fdtype_ops memfd_secret_dump_ops;
+extern struct collect_image_info memfd_secret_cinfo;
+
+#ifdef CONFIG_HAS_MEMFD_SECRET
```

Comment:
As I've mentioned earlier, we can detect availability of `memfd_secret` in runtime. If `syscall(__NR_memfd_secret, 0)` returns `-ENOSYS`, the system call is not available. Otherwise secretmem is enabled.

Reply:
done, thanks!